tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -43,35 +42,10 @@ def rhev():
     )
     rhev.cluster_id = rhev.rhv_api.get_cluster(rhev.cluster).id
     rhev.storage_id = rhev.rhv_api.get_storage_domain(rhev.storage_domain).id
-    if is_open('BZ:1685949'):
-        dc = rhev.rhv_api._data_centers_service.list(search=f'name={rhev.datacenter}')[0]
-        dc = rhev.rhv_api._data_centers_service.data_center_service(dc.id)
-        rhev.quota = dc.quotas_service().list()[0].id
-    else:
-        rhev.quota = 'Default'
+    rhev.quota = 'Default'
     return rhev
 
 
-@pytest.fixture(scope='module')
-def provisioning(module_org, module_location):
```

Comment:
There seems one more test `test_positive_provision_rhev_image_based_and_disassociate` using these fixtures in the same test module, you could update that as well?